deployment/kubernetes/ktranslate-nr.yaml

```diff
@@ -104,23 +109,28 @@ kind: ConfigMap
 metadata:
   name: ktranslate-config
 data:
-  snmp.yml: |
+  snmp-base.yaml: |
     devices:
-      switch:
-        device_name: switch
-        device_ip: 10.10.0.10
-        flow_only: true
-        user_tags: {}
     trap:
       listen: 0.0.0.0:1620
       community: hello
```

Comment:
If we are in here making changes, the community string on the listener does not appear to do anything, it does not filter incoming messages that use a different community and I have used it successfully without this line in the past.

Reply:
fair point - @314 - are we clear to remove `trap.community` as well? I think so based on [this bit](https://github.com/kentik/ktranslate/blob/729ff43fb7b949b60f7cfad3a1e474dfeaf80178/pkg/inputs/snmp/traps/traps.go#L62-L63)

```go
	if conf.Trap.Community == "" {
		conf.Trap.Community = "hello"
	}
```

Reply:
I think so too. Fine to remove.